### PR TITLE
[apiap] Extract Policies to its own page

### PR DESCRIPTION
**Pull request by hallelujah**
Fixes THREESCALE-3450

### CODE REVIEW

File: app/controllers/api/policies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  protected
+
+  def proxy_params
+    params.require(:proxy).permit(:policies_config)
+  end
+
+  def check_permission!
+    provider_can_use! :api_as_product
+    authorize! :edit, service
```
Comment (hallelujah) [inline app/controllers/api/policies_controller.rb]: We have the service permission (which does not really work well sadly), Just thinking about the future

**Comment by guicassolato** [comment]
I fixed https://github.com/3scale/porta/blob/8b63356fc1e8c105de40445a06b19f9fcc7c7f06/features/old/menu/api_menu.feature#L59-L67 in https://github.com/3scale/porta/pull/1230. You'll probably gonna need it here.

File: app/javascript/src/Policies/styles/policies.scss [hunk 2]
```diff
@@ -15,12 +15,6 @@ $error-color: #c00;
   line-height: 1.5;
   padding-bottom: 2rem;
 
-  .PolicyChain, .PolicyRegistryList, .PolicyConfiguration {
```
Comment (guicassolato) [inline app/javascript/src/Policies/styles/policies.scss]: I think the partial will continue to be rendered in the integration form for non-APIAP accounts. Won't this change affect the way those users see the Policies section in the page?

Comment (guicassolato) [reply app/javascript/src/Policies/styles/policies.scss]: Even for APIAP-viewers some minor fixes need to be done:

Screen Shot 2019-09-25 at 1 17 30 PM

**Comment by thomasmaas** [comment]
@hallelujah did you think about that policies page should not appear in nav when user has selected Istio?

**Pull request merged**